packages/shared/src/components/cards/common/SourceButton.tsx

```diff
@@ -1,29 +1,61 @@
 import type { CSSProperties, ReactElement } from 'react';
-import React from 'react';
+import React, { useMemo } from 'react';
+import dynamic from 'next/dynamic';
 import type { TooltipPosition } from '../../tooltips/BaseTooltipContainer';
 import { LinkWithTooltip } from '../../tooltips/LinkWithTooltip';
 import { ProfileImageLink } from '../../profile/ProfileImageLink';
 import { ProfileImageSize, ProfilePicture } from '../../ProfilePicture';
-import type { Source } from '../../../graphql/sources';
+import type { SourceTooltip } from '../../../graphql/sources';
 import { useFeedPreviewMode } from '../../../hooks';
+import { Origin } from '../../../lib/log';
+
+const SquadEntityCard = dynamic(
+  /* webpackChunkName: "SquadEntityCard" */ () =>
```

Comment:
Just to follow the current convention. I think we use camelCase.
```suggestion
  /* webpackChunkName: "squadEntityCard" */ () =>
```